crates/proof-of-sql/src/sql/postprocessing/select_expr.rs

```diff
@@ -0,0 +1,44 @@
+use super::{PostprocessingEvaluator, PostprocessingResult, PostprocessingStep};
+use crate::base::{
+    database::{OwnedColumn, OwnedTable},
+    scalar::Scalar,
+};
+use indexmap::IndexMap;
+use proof_of_sql_parser::{intermediate_ast::AliasedResultExpr, Identifier};
+use serde::{Deserialize, Serialize};
+
+/// The select expression used to select, reorder, and apply alias transformations
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct SelectExpr<S: Scalar> {
+    /// The aliased result expressions we select
+    aliased_result_exprs: Vec<AliasedResultExpr>,
+    _phantom: core::marker::PhantomData<S>,
+}
```

Comment:
This should work too, right? We might need it later, but maybe not.
```suggestion
pub struct SelectExpr {
    /// The aliased result expressions we select
    aliased_result_exprs: Vec<AliasedResultExpr>,
}
```